integration/client/client_unix_test.go

```diff
@@ -30,7 +30,7 @@ var (
 	testImage             = images.Get(images.BusyBox)
 	testMultiLayeredImage = images.Get(images.VolumeCopyUp)
 	shortCommand          = withProcessArgs("true")
-	longCommand           = withProcessArgs("/bin/sh", "-c", "while true; do sleep 1; done")
+	longCommand           = withProcessArgs("/bin/sh", "-c", "while true; do sleep 1d; done")
```

Comment:
Side note: the `longCommand` applies to `unix` build flags.

Reply:
What about this instead?

```suggestion
	longCommand           = withProcessArgs("/bin/sh", "-c", "sleep inf")
```